src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -151,7 +152,7 @@ def create_request(
         # Create the Search Object
         es_search = Search(
             using=es_client,
-            index=os.getenv(index, 'browser_files_dev'))
+            index=azul_config.es_index_name(index) or 'azul_files_dev')
```

Comment:
It's ok to eliminate the fallback. There is no point to it. If we wanted a fallback we would implement it in es_index_name.

Here and everywhere else.